libs/client/my-stuff/src/lib/views/prose-form/prose-form.component.html

```diff
@@ -55,11 +55,11 @@ <h3>{{ formTitle }}</h3>
                     </div>
                 </div>
 
-                <ng-container *ngIf="proseForm.controls.category.value === categories.Fanwork">
+                <ng-container *ngIf="proseForm.controls.category.value === categories.Fanwork && tagsEnabled">
```

Comment:
we'll have to remember to remove these checks once we flip the feature flag on...